app/views/layouts/_center_div_no_listnav.html.haml

```diff
@@ -8,7 +8,7 @@
         .row.toolbar-pf#toolbar
           .col-sm-12
             = render :partial => "layouts/angular/toolbar"
-      .row#main-content.miq-layout-center_div_no_listnav{:class => @lastaction == "show_dashboard" ? 'miq-sand-paper' : ''}
```

Comment:
Intentionally changing `miq-sand-paper` into `miq-body`?

If so, we have...

```
app/assets/javascripts/miq_explorer.js
156:  mainContent.removeClass('miq-sand-paper');

app/assets/javascripts/controllers/report_data_controller.js
412:    angular.element(mainContent).removeClass('miq-sand-paper');
417:      angular.element(mainContent).addClass('miq-sand-paper');

app/views/service/explorer.html.haml
14:        mainContent.removeClass('miq-sand-paper');
```

which probably also need to be updated.

Reply:
> Intentionally changing miq-sand-paper into miq-body? 

Yes, intentionally, miq-sandpaper has some other properties we don't want on those screens